deployment-template/eks/otel-fargate-container-insights.yaml

```diff
@@ -0,0 +1,552 @@
+        eks_fargate_container_insights_enabled: true
+        parse_json_encoded_attr_values: ["kubernetes"]
+        dimension_rollup_option: NoDimensionRollup
+        metric_declarations:
+          - dimensions: [ [ClusterName, LaunchType], [ClusterName, Namespace, LaunchType], [ClusterName, Namespace, PodName, LaunchType]]
+            metric_name_selectors:
+              - pod_cpu_utilization_over_pod_limit
+              - pod_cpu_usage_total
+              - pod_cpu_limit
+              - pod_memory_utilization_over_pod_limit
+              - pod_memory_working_set
+              - pod_memory_limit
+              - pod_network_rx_bytes
+              - pod_network_tx_bytes
+
+      logging:
+        loglevel: debug
+
+    extensions:
+      health_check:
+
+    service:
+      pipelines:
+        metrics:
+          receivers: [prometheus]
+          processors: [metricstransform/label_1, resourcedetection, metricstransform/rename, filter, cumulativetodelta, deltatorate, experimental_metricsgeneration/1, experimental_metricsgeneration/2, metricstransform/label_2, batch]
+          exporters: [logging, awsemf]
+      extensions: [health_check]
+
+# configure the service and the collector as a StatefulSet
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: adot-collector-service
+  namespace: fargate-container-insights
+  labels:
+    app: aws-adot
+    component: adot-collector
+spec:
+  ports:
+    - name: metrics # default endpoint for querying metrics.
+      port: 8888
+  selector:
+    component: adot-collector
+  type: ClusterIP
+
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: adot-collector
+  namespace: fargate-container-insights
+  labels:
+    app: aws-adot
+    component: adot-collector
+spec:
+  selector:
+    matchLabels:
+      app: aws-adot
+      component: adot-collector
+  serviceName: adot-collector-service
+  template:
+    metadata:
+      labels:
+        app: aws-adot
+        component: adot-collector
+    spec:
+      serviceAccountName: adot-collector
+      securityContext:
+        fsGroup: 65534
+      containers:
+        - image: amazon/aws-otel-collector:latest
+          name: adot-collector
+          imagePullPolicy: Always
+          command:
+            - "/awscollector"
+            - "--config=/conf/adot-collector-config.yaml"
+          env:
+            - name: OTEL_RESOURCE_ATTRIBUTES
+              value: "ClusterName=YOUR-EKS-CLUSTER-NAME"
+          resources:
+            limits:
+              cpu: 2
+              memory: 2Gi
+            requests:
+              cpu: 200m
+              memory: 400Mi
+          volumeMounts:
+            - name: adot-collector-config-volume
+              mountPath: /conf
+      volumes:
+        - configMap:
+            name: adot-collector-config
+            items:
+              - key: adot-collector-config
+                path: adot-collector-config.yaml
+          name: adot-collector-config-volume
+---
```

Comment:
Needs `\n` at EOF.

Reply:
got it

Reply:
i added a new line at EOF in the latest commit but seems like it still shows the same in the pr